hudi-client/hudi-client-common/pom.xml

```diff
@@ -43,6 +43,16 @@
       <artifactId>hudi-common</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hudi</groupId>
+      <artifactId>hudi-io</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
Is this needed, given `hudi-common` module pulls in `hudi-io` dependency transitively?

Reply:
That's what I initially assumed, but I was getting some class not found issues  and this fixed it